### PR TITLE
Tests for archsoot.py

### PR DESCRIPTION
**Pull request by anupriya-gupta**

### CODE REVIEW

File: tests/test_arch_soot.py [hunk 1]
```diff
@@ -0,0 +1,430 @@
+from archinfo.arch import Endness
+from nose.tools import raises
+from archinfo.arch_soot import SootAddressDescriptor, SootMethodDescriptor, SootAddressTerminator, SootFieldDescriptor, SootClassDescriptor, SootNullConstant, SootArgument, ArchSoot
+import nose.tools
```
Comment (twizmwazin) [inline tests/test_arch_soot.py]: These should probably be reordered. https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement might be helpful.

File: tests/test_import.py [hunk 2]
```diff
@@ -1 +1,2 @@
-import archinfo
+def test_import():
```
Comment (twizmwazin) [inline tests/test_import.py]: I think this is the wrong PR for this change

File: tests/test_register.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+def test_register():
```
Comment (twizmwazin) [inline tests/test_register.py]: I think this is the wrong PR for this change

File: tests/test_arch_soot.py (hunk 1 above)
Comment (anupriya-gupta) [reply tests/test_arch_soot.py]: Okay. I'll do that.

**Pull request closed without merging**